apm-lambda-extension/Makefile

```
@@ -57,7 +61,11 @@ endif
 	GOARCH=${GOARCH} make zip
 	$(MAKE) publish
 zip:
-	cd bin && rm -f extension.zip || true && zip -r extension.zip extensions NOTICE.txt dependencies.asciidoc && cp extension.zip ${GOARCH}.zip
+	cd bin \
+	&& rm -f extension.zip || true \
```

```suggestion
	&& rm -f extension.zip \
```

pretty sure that's redundant with the `-f` flag?